Without this patch Lablgtk treats warnings as errors.

This causes the build to fail with our currently packed ocaml version:

	File "gdk.ml", line 346, characters 2-55:
	346 |   external create : len:int -> t = "ml_point_array_new"
		^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
	Error (warning 61): This primitive declaration uses type t, which is unannotated and
	unboxable. The representation of such types may change in future
	versions. You should annotate the declaration of t with [@@boxed]
	or [@@unboxed].

diff -upr lablgtk-2.18.6.orig/src/Makefile lablgtk-2.18.6/src/Makefile
--- lablgtk-2.18.6.orig/src/Makefile	2019-12-05 13:29:04.763064450 +0100
+++ lablgtk-2.18.6/src/Makefile	2019-12-05 13:29:20.569735224 +0100
@@ -1,6 +1,6 @@
 # Makefile for lablgtk.
 
-COMPILER = $(CAMLC) $(MLFLAGS) $(MLBYTEFLAGS) -w s-3+52 -warn-error A-52 -c
+COMPILER = $(CAMLC) $(MLFLAGS) $(MLBYTEFLAGS) -w s-3+52 -c
 LINKER = $(CAMLC) $(MLFLAGS) $(MLBYTEFLAGS)
 COMPOPT = $(CAMLOPT) $(MLFLAGS) -w s -c
 LINKOPT = $(CAMLOPT) $(MLFLAGS)
